src/main/java/org/diego/tutorial/car/model/Car.java

```diff
@@ -41,10 +42,13 @@
 	@Column(nullable = false)
 	@Temporal(TemporalType.TIMESTAMP)
 	private Date lastUpdated;
-	
+
 	@Transient // Links are not stored in database
 	private List<Link> links = new ArrayList<Link>();
 
+	@DefaultValue(value = "false")
+	private boolean checked;

```

Comment:
is it not a db field on purpose?

Reply:
It is a db field. On the database the field is created.

Reply:
In the other fields, I used @column annotation to specify that they should not be null. But a boolean cannot be null, just true/false, and I do not need to change the field name of the database, checked is fine. So I think that there is no need to add the @column annotation.

Reply:
sounds good to me